.travis.yml

```diff
@@ -12,8 +12,8 @@ install:
 script:
   - set -e
   - ./scripts/check-pristine-state package-lock.json
+  - (set -o pipefail; npm test |& tee test.log);
```

Comment:
* Keep test output in a file so Danger can parse it.
* Run tests before Danger so it has access the the test logs